claims/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+name = "claims"
```

Comment:
Remember (from the other conversations) that crates live in a global flat namespace, so they need to have globally unique names. In particular, it looks like this name is in fact already taken: https://crates.io/crates/claims . This caused us actual issues in the past, in which cargo would sometimes start depending on the external crate without a way of overriding it.

Please use a name like oak_transparent_claims or at least oak_claims

Reply:
Or `oak_transparency_claims`

Reply:
Changed the crate name to `oak_transparency_claims`.